test/evergreen/coverage-report.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+set -o errexit
+set -o verbose
+
+# Check if enough arguments are given, pr_args can be an empty string for patch builds.
+if [ $# -lt 3 ]; then
```

Comment:
I would just combine both if conditions in this case.

Reply:
if statements combined in [eb83923](https://github.com/wiredtiger/wiredtiger/pull/10915/commits/eb83923db61711b2bf4f900af80ff204588617ee)